src/sas/qtgui/Perspectives/Corfunc/UI/CorfuncPanel.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1157</width>
-    <height>885</height>
+    <width>1113</width>
+    <height>746</height>

```

Comment:
There are a number of changes to the widget sizes. Likely introduced while modifying the UI files in Qt Creator. The UI files are xml, so any text editor will allow you to change them back to their original values.

Reply:
Yes, this was done by accident. I will change them back of course.